samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -108,17 +108,6 @@ public StreamProcessor(int processorId, Config config, Map<String, MetricsReport
     this(processorId, config, customMetricsReporters, (Object) streamTaskFactory);
   }
 
-  /**
-   * Same as {@link #StreamProcessor(int, Config, Map, AsyncStreamTaskFactory)}, except task instances are created
-   * using the "task.class" configuration instead of a task factory.
-   * @param processorId - this processor Id
-   * @param config - config
-   * @param customMetricsReporters metrics
-   */
```

Comment:
I see. That sounds good. So, I am guessing the taskfactory creation will be removed from SamzaContainer as well??